packages/core/admin/admin/src/pages/Home/components/ContentManagerWidgets.tsx

```diff
@@ -0,0 +1,154 @@
+        {formatMessage({
+          id: `content-manager.containers.List.${status}`,
+          defaultMessage: capitalise(status),
+        })}
+      </Typography>
+    </Status>
+  );
+};
+
+/* -------------------------------------------------------------------------------------------------
+ * LastEditedWidget
+ * -----------------------------------------------------------------------------------------------*/
+
+const LastEditedContent = () => {
+  const { formatMessage } = useIntl();
+  const { trackUsage } = useTracking();
+  const navigate = useNavigate();
+  const { data, isLoading, error } = useGetRecentDocumentsQuery({ action: 'update' });
+
+  if (isLoading) {
+    return <Widget.Loading />;
+  }
+
+  if (error) {
+    return <Widget.Error />;
+  }
+
+  if (data?.length === 0) {
+    return (
+      <Widget.NoData>
+        {formatMessage({
+          id: 'content-manager.widget.last-edited.no-data',
+          defaultMessage: 'No edited entry',
+        })}
+      </Widget.NoData>
+    );
+  }
+
+  const handleRowClick = (document: RecentDocument) => () => {
+    trackUsage('willEditEntryFromHome');
+    const link = getEditViewLink(document);
+    navigate(link);
+  };
+
+  return (
+    <Table colCount={5} rowCount={data?.length ?? 0}>
+      <Tbody>
+        {data?.map((document) => (
+          <Tr onClick={handleRowClick(document)} key={document.documentId}>
+            <Td>
+              <CellTypography variant="omega" textColor="neutral800">
+                {document.title}
+              </CellTypography>
+            </Td>
+            <Td>
+              <CellTypography variant="omega" textColor="neutral600">
+                {document.kind === 'singleType'
+                  ? formatMessage({
+                      id: 'content-manager.widget.last-edited.single-type',
+                      defaultMessage: 'Single-Type',
+                    })
+                  : // TODO check how to localize display name
+                    document.contentTypeDisplayName}
+              </CellTypography>
+            </Td>
+            <Td>
+              <Box display="inline-block">
+                <DocumentStatus status={document.status} />
+              </Box>
+            </Td>
+            <Td>
+              <Typography textColor="neutral600">
+                <RelativeTime timestamp={new Date(document.updatedAt)} />
+              </Typography>
+            </Td>
+            <Td onClick={(e) => e.stopPropagation()}>
+              <Box display="inline-block">
+                <IconButton
+                  tag={Link}
+                  to={getEditViewLink(document)}
+                  onClick={() => trackUsage('willEditEntryFromHome')}
+                  label={formatMessage({
+                    id: 'content-manager.actions.edit.label',
+                    defaultMessage: 'Edit',
+                  })}
+                  variant="ghost"
+                >
+                  <Pencil />
+                </IconButton>
```

Comment:
Shouldn't we only show this if the user has the permissions to access the document?

Reply:
Or no wait we only send permitted documents, right?

Reply:
yes the backend should do all the permission checks for us